ansible/packer/orka.pkr.hcl

```diff
@@ -0,0 +1,78 @@
+packer {
+  required_plugins {
+    macstadium-orka = {
+      source  = "github.com/macstadium/macstadium-orka"
+      version = "~>3"
+    }
+    ansible = {
+      source  = "github.com/hashicorp/ansible"
+      version = "~> 1"
+    }
+  }
+}
+
+variable "ORKA_TOKEN" {
+  default = env("ORKA_TOKEN")
+}
+
+variable "ORKA_ENDPOINT" {
+  default = "http://10.221.188.20"
```

Comment:
Same as before

Reply:
it's the Orka endpoint (documented in Bitwarden)